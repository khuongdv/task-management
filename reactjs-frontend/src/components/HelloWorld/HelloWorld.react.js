import React from 'react';

class HelloWorld extends React.Component {
  render() {
    return (
      <a className="hello-world" href={this.props.url || "#"}>Hello, world</a>
    );
  }
}

export default HelloWorld;
