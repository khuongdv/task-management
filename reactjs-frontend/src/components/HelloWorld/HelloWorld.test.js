import React from 'react';
import HelloWorld from './HelloWorld.react';
import renderer from 'react-test-renderer';

it('helloworld', () => {
  const component = renderer.create(
    <HelloWorld url="hello.com"/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
