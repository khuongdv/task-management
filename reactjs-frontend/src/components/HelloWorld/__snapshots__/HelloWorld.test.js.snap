exports[`test helloworld 1`] = `
<a
  className="hello-world"
  href="hello.com">
  Hello, world
</a>
`;
